### DISCUSSION TITLE
when i try to create an account with git hub iit says account has been blocked

### DISCUSSION THREAD

**Discussion by austinewalking** [vercel/community#4340] [Account Issue] [answered]
### Question

why am i getting this error

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by austinewalking** [comment]
What I need is help..
I have send multiple messages to vercel support team yet they haven't
resolved it

**Reply by amyegan** [reply]
The team is facing an unusually high number of cases. Tickets are processed by the order in which they are received, so it will take some time for the team to respond to all of the cases that were previously opened before they reach yours. 🙇 

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time

**Comment by austinewalking** [comment]
They haven't fixed it ...and it is frustrating
I have to host my site

**Comment by austinewalking** [comment]
Why haven't this not been fixed
